.github/workflows/build_and_test.yaml

```diff
@@ -157,7 +157,6 @@ jobs:
       - if: ${{ env.ARCH == 'x86_64' }} 
         name: Run VM script dependencies
         run: |
-          brew update; brew upgrade qemu
```

Comment:
Yep, this was added since there was an update to qemu which was not picked up by our runner, which broke the build... Probably not needed now!

Reply:
Yes I guess we were suffering MacOS runner upgrades. I remember there used to be a very old qemu version when we first started using it, now I see it comes with Qemu v8 by default.